### ISSUE TITLE
Look at top card, draw a card, drag face down will be revealed even though you dont know that card

### ISSUE THREAD

**Issue by MultiDragon** [Cockatrice/Cockatrice#4670]
**System Information:**
```
Client Version: 2.8.0 (2021-01-26)
Client Operating System: EndeavourOS
Build Architecture: 64-bit
Qt Version: 5.15.4
System Locale: en_US
Install Mode: Standard
```

_______________________________________________________________________________________
Looking at the top X cards of your library and then dragging cards on the battlefield face down reveals them, even if you already drew the X cards in question. This is fixed if you close your "looking at library" popup. A video of the bug is attached.

https://user-images.githubusercontent.com/52964431/189984985-7881671c-8f85-4d2b-b4d8-4dcce0b0ac06.mp4

_______________________________________________________________________________________

**Steps to reproduce:**
 - Press Ctrl+W, 1 to look at the top card
 - Press Ctrl+D to draw that card
 - Play top card face down
 - Top card is revealed in chatlog

**Comment by MultiDragon** [comment]
I hope it's not a duplicate, I did search for all "Bugs with face down" and did not find but could be named differently

**Comment by ebbit1q** [comment]
I'm amazed noone noticed this bug before, I will try to fix this asap

**Comment by ebbit1q** [comment]
this is also partially fixed already in #4491, it was just when drawing that it still had the unintentional behavior
